services/socketService.js

```diff
@@ -39,6 +55,8 @@ const connection = function (socket) {
 
   socket.on('joinChat', joinChat);
 
```

Comment:
I think we should remove these blank lines.

Reply:
Sure